### PR TITLE
Implement HTTP Archive query to assess usage of block themes.

### LINKED ISSUE #9: Query to assess usage of block themes
* Create HTTP Archive query to get **% of WordPress sites that include a `div.wp-site-blocks` in their markup**

### PR DESCRIPTION
**Pull request by felixarntz**
Fixes #9

This query identifies sites using a block theme by checking for the `<div class="wp-site-blocks">` tag which is present in all such sites, as controlled by WordPress core.

## Query results

Row | client | with_block_theme | total_wp_sites | pct_total | wp_gte_59
-- | -- | -- | -- | -- | --
1 | desktop | 7190 | 4241264 | 0.0016952493407625651 | 3393158
2 | mobile | 13074 | 5703734 | 0.0022921826298351222 | 4471610

Based on November 2022 dataset

### CODE REVIEW

**Comment by felixarntz** [comment]
@vishalkakadiya @ankitrox Could you please review this? It's relevant to several analysis, but particularly was called out for the lazy-loading research and the web fonts API research.

Overall, this data shows that, at least as of November, only ~0.2% of all WordPress sites were using a block theme.

**Comment by vishalkakadiya** [comment]
@felixarntz The query looks good to me though I haven't run it as it is processing more than **1TB** of data. Thank you!

**Review by vishalkakadiya — APPROVED** [review]

File: sql/2023/01/block-theme-usage.sql [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  (_TABLE_SUFFIX, url)
+JOIN (
+  SELECT
+    _TABLE_SUFFIX,
+    COUNT(DISTINCT IF (info = '' OR CAST(REGEXP_EXTRACT(info, r'^(\d+\.\d+)') AS FLOAT64) >= 5.9, url, NULL)) AS wp_gte_59,
+    COUNT(DISTINCT url) AS total_wp_sites
+  FROM
+    `httparchive.technologies.2022_11_01_*`
+  WHERE
+    app = "WordPress"
```
Comment (mukeshpanchal27) [inline sql/2023/01/block-theme-usage.sql:39]: Use `category = 'CMS'` to remove duplicate entry.

Ie. `SELECT * FROM httparchive.technologies.2022_12_01_desktop WHERE app = "WordPress" AND url ='http://alvkullen.se/'` show the website in `CMS` and 'Blogs` categories.

```suggestion
    app = "WordPress"
    AND category = 'CMS'
```

Comment (mukeshpanchal27) [reply sql/2023/01/block-theme-usage.sql:39]: I found that you used `DISTINCT url` for count but not sure if above affect anything in result or not as i'm enable to run the query.

Comment (felixarntz) [reply sql/2023/01/block-theme-usage.sql:39]: @mukeshpanchal27 See my reply on Slack: There are no duplicate entries when using `DISTINCT`, so I think this isn't needed here.

Comment (mukeshpanchal27) [reply sql/2023/01/block-theme-usage.sql:39]: Thanks for the clarification.

**Review by mukeshpanchal27 — DISMISSED** [review]
Thanks @felixarntz, LGTM! Just needs to resolve the conflict.

**Review by mukeshpanchal27 — APPROVED** [review]

**Review by vishalkakadiya — APPROVED** [review]

**Comment by vishalkakadiya** [comment]
@felixarntz Approved now!

**Comment by felixarntz** [comment]
Thanks!

**Comment by felixarntz** [comment]
@mukeshpanchal27 @vishalkakadiya FYI see https://github.com/HTTPArchive/custom-metrics/issues/61 for adding a custom metric to HTTP Archive so that we will be able to run a much simpler version of this query in the future.

**Pull request merged**